docs/src/pages/demos/DemoAnimateHeight.js

```diff
@@ -40,7 +39,8 @@ class Slide4 extends PureComponent {
   };
 
   componentDidUpdate() {
-    this.context.swipeableViews.slideUpdateHeight();
+    const { slideUpdateHeight } = useContext(swipeableViewsContext);
```

Comment:
How does it work? useContext look like a hook inside a class component lifecycle. I I was expecting https://reactjs.org/docs/context.html#classcontexttype.

Reply:
+1 @AlbertCarreras Usage of hooks is forbidden inside class components. Would you mind updating your PR with the suggestion brought up by @oliviertassinari ?